src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
At the moment this library focuses on cookie based sessions. I'm happy to merge the other changes to make it possible for you to use with WebSockets, but please remove this one and just disable cookies in your application